en/advanced_config/racer_setup.md

```diff
@@ -0,0 +1,200 @@
+Left-right balance is usually not a problem, but front-back balance may be.
+You can move the battery until it is correct and mark it on the frame so you will always place it correctly.
+
+> **Note** The integral term can account for an imbalanced setup, and a custom mixer can do that even better. However it is best to fix any imbalance as part of the vehicle setup.
+
+## Software Setup
+
+After having built the racer, you will need to configure the software.
+Go through the [Basic Configuration Guide](../config/README.md) and choose the
+[Generic 250 Racer](../airframes/airframe_reference.md#copter_quadrotor_x_generic_250_racer)
+airframe, which already sets some racer-specific parameters.
+
+These parameters are important:
+- Enable One-Shot by setting [PWM_RATE](../advanced_config/parameter_reference.md#PWM_RATE) to 0.
+- Enable `SYS_FMU_TASK` (explained [below](#control_latency)).
+- Set the maximum roll-, pitch- and yaw rates for Manual/Stabilized mode as
+  desired: [MC_ROLLRATE_MAX](../advanced_config/parameter_reference.md#MC_ROLLRATE_MAX), [MC_PITCHRATE_MAX](../advanced_config/parameter_reference.md#MC_PITCHRATE_MAX) and [MC_YAWRATE_MAX](../advanced_config/parameter_reference.md#MC_YAWRATE_MAX).
+  The maximum tilt angle is configured with [MPC_MAN_TILT_MAX](../advanced_config/parameter_reference.md#MPC_MAN_TILT_MAX).
+- The minimum thrust [MPC_MANTHR_MIN](../advanced_config/parameter_reference.md#MPC_MANTHR_MIN) should be set to 0.
+- Disable RC input filtering by setting [RC_FLT_CUTOFF](../advanced_config/parameter_reference.md#RC_FLT_CUTOFF) to 0.
+
+### Estimator
+
+If you use a GPS you can skip this section and use the default estimator.
+Otherwise you should switch to the Q attitude estimator, which works without a magnetometer or barometer. 
+
+To select it, set [SYS_MC_EST_GROUP](../advanced_config/parameter_reference.md#SYS_MC_EST_GROUP) to 1, and change the
+following parameters:
+- Set [SYS_HAS_MAG](../advanced_config/parameter_reference.md#SYS_HAS_MAG) to 0 if the system does not have a magnetometer.
+- Set [SYS_HAS_BARO](../advanced_config/parameter_reference.md#SYS_HAS_BARO) to 0 if the system does not have a barometer.
+- Configure the Q estimator: set [ATT_ACC_COMP](../advanced_config/parameter_reference.md#ATT_ACC_COMP) to 0, [ATT_W_ACC](../advanced_config/parameter_reference.md#ATT_W_ACC) to 0.4 and
+  [ATT_W_GYRO_BIAS](../advanced_config/parameter_reference.md#ATT_W_GYRO_BIAS) to 0. You can tune these later if you wish.
+
+### Failsafe
+
+Configure [RC loss and low battery failsafe](../config/safety.md). 
+If you do not use a GPS, set the failsafe to **Lockdown**, which turns off the motors. 
+Test RC loss on the bench without props attached by turning off the remote when the vehicle is armed.
+
+Make sure to assign a [kill switch](../config/flight_mode.md#kill-switch) or an [arming switch](#arming_switch).Test it and train to use it!
+
+#### Arm Switch {#arming_switch}
```

Comment:
@bkueng When would you use a kill switch and when an arming switch - what is the effective difference for a user? There is a practical difference that there is an option to select kill switch as a flight mode in QGC setup but not arming switch.

So I think this might be more useful generically than racers, but it is not clear where it should go. I have the same problem with kill switch, which currently gets referenced as a flight mode you can select [here](https://docs.px4.io/en/config/flight_mode.html#kill-switch). I am aware it is not a flight mode, but it made sense to put it there as it was set as a flight mode.
Proposal is to move docs for kill switch and arming switch into the Safety page here: https://docs.px4.io/en/config/safety.html - then link to that from everywhere.  Thoughts?

Reply:
I've found there's a fair amount of user confusion over the kill switch and arming switch.

Reply:
@dagar Me too. Can you resolve it - ie explain the effective difference?

Reply:
Kill-switch: it is a safety switch that you can always trigger and it will always immediately kill the motors. Whereas the arming switch will have the same effect in manual or acro, but for example in position mode, it will not allow you to disarm while the land detector did not detect landing yet (while the vehicle is still in-air).

Also the armings logic is more complex (depends on flight mode, etc.), and thus if you are doing low-level FC software changes a kill-switch just gives you more confidence in making sure the system turns off once you trigger it.

> There is a practical difference that there is an option to select kill switch as a flight mode in QGC setup but not arming switch.

Yes the arming switch should be exposed as well in QGC. But I first wanted to change its logic a bit (https://github.com/PX4/Firmware/pull/9622). 

> Proposal is to move docs for kill switch and arming switch into the Safety page here: https://docs.px4.io/en/config/safety.html - then link to that from everywhere. Thoughts?

Sounds good. It is not limited to racers, but they are particularly interested in this.